<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProcessingWidget</class>
 <widget class="QFrame" name="ProcessingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>370</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>370</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame
{
background:#0E1417;
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="CArmBothwaySlider" name="contsSlider">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>200</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal{
height: 4px;
background-color:rgba(66,80,96,1) ;
border: 1px solid #566576;
border-radius: 0px;
padding-left:-1px;
padding right:-1px;}
        
QSlider::sub-page:horizontal{
height: 4px;
background: transparent;
border: 1px solid transparent;
border-radius: 2px;
}
  
QSlider::handle:horizontal
{
    width: 22px;
    background-color:rgba(255,255,255,1);
    margin-top: -10px;
    margin-bottom: -10px;
border-radius: 5px;
}</string>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="CArmBothwaySlider" name="brtSlider">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>108</y>
     <width>200</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal{
height: 4px;
background-color:rgba(66,80,96,1) ;
border: 1px solid #566576;
border-radius: 0px;
padding-left:-1px;
padding right:-1px;}
        
QSlider::sub-page:horizontal{
height: 4px;
background: transparent;
border: 1px solid transparent;
border-radius: 2px;
}
  
QSlider::handle:horizontal
{
    width: 22px;
    background-color:rgba(255,255,255,1);
    margin-top: -10px;
    margin-bottom: -10px;
border-radius: 5px;
}</string>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
image:url(&quot;:/FunctionalWidget/FunctionalWidget/Review/Resource/Contrasticon.png&quot;);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="contsAuto">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>70</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>36</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>36</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
background:rgba(11,209,140,0.24);
border-radius:1px;
opacity:0.24;
color:rgba(11,209,140,1);
}</string>
   </property>
   <property name="text">
    <string>自动</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>108</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
image:url(&quot;:/FunctionalWidget/FunctionalWidget/Review/Resource/Bricon.png&quot;);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="brtAuto">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>138</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>36</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>36</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
background:rgba(11,209,140,0.24);
border-radius:1px;
opacity:0.24;
color:rgba(11,209,140,1);
}</string>
   </property>
   <property name="text">
    <string>自动</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="shpSlider">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>200</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal{
height: 4px;
background-color:rgba(66,80,96,1) ;
border: 1px solid #566576;
border-radius: 0px;
padding-left:-1px;
padding right:-1px;}
        
QSlider::sub-page:horizontal{
height: 4px;
background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 : 1, stop : 0 #0091FF, stop:1 #0091FF);
background: qlineargradient(x1 : 0, y1 : 0.2, x2 : 1, y2 : 1, stop : 0 #0091FF, stop:1 #0091FF);
border: 1px solid #4A708B;
border-radius: 2px;
}
  
QSlider::handle:horizontal
{
    width: 22px;
    background-color:rgba(255,255,255,1);
    margin-top: -10px;
    margin-bottom: -10px;
border-radius: 5px;
}</string>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
image:url(:/FunctionalWidget/FunctionalWidget/Review/Resource/Shicon.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="dnsSlider">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>234</y>
     <width>200</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove:horizontal{
height: 4px;
background-color:rgba(66,80,96,1) ;
border: 1px solid #566576;
border-radius: 0px;
padding-left:-1px;
padding right:-1px;}
        
QSlider::sub-page:horizontal{
height: 4px;
background: qlineargradient(x1 : 0, y1 : 0, x2 : 0, y2 : 1, stop : 0 #0091FF, stop:1 #0091FF);
background: qlineargradient(x1 : 0, y1 : 0.2, x2 : 1, y2 : 1, stop : 0 #0091FF, stop:1 #0091FF);
border: 1px solid #4A708B;
border-radius: 2px;
}
  
QSlider::handle:horizontal
{
    width: 22px;
    background-color:rgba(255,255,255,1);
    margin-top: -10px;
    margin-bottom: -10px;
border-radius: 5px;
}</string>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>234</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
image:url(:/FunctionalWidget/FunctionalWidget/Review/Resource/Densicon.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>296</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
image:url(:/FunctionalWidget/FunctionalWidget/Review/Resource/Mticon.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="mt0Btn">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>296</y>
     <width>45</width>
     <height>28</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>28</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>28</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton
{
background-color:rgba(255,255,255,0.1);
border-radius:0px;
font:16px Microsoft YaHei;
color:rgba(255,255,255,0.4);
border: 1px solid transparent;
}

QToolButton:checked
{
background-color:rgba(0,145,255,1); 
border-radius:0px 0px 0px 2px;
font:16px Microsoft YaHei; 
color:rgba(255,255,255,1);
border: 1px solid transparent;
}
</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="mt1Btn">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>296</y>
     <width>47</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>47</width>
     <height>28</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>28</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton
{
background-color:rgba(255,255,255,0.1);
border-radius:0px;
font:16px Microsoft YaHei;
color:rgba(255,255,255,0.4);
border:none;
}

QToolButton:checked
{
background-color:rgba(0,145,255,1); 
border-radius:0px 0px 0px 2px;
font:16px Microsoft YaHei; 
color:rgba(255,255,255,1);
border:none;
}
</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="mt2Btn">
   <property name="geometry">
    <rect>
     <x>183</x>
     <y>296</y>
     <width>47</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>47</width>
     <height>28</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>28</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton
{
background-color:rgba(255,255,255,0.1);
border-radius:0px;
font:16px Microsoft YaHei;
color:rgba(255,255,255,0.4);
border:none;
}

QToolButton:checked
{
background-color:rgba(0,145,255,1); 
border-radius:0px 0px 0px 2px;
font:16px Microsoft YaHei; 
color:rgba(255,255,255,1);
border:none;
}
</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="mt3Btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>296</y>
     <width>47</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>47</width>
     <height>28</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>28</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton
{
background-color:rgba(255,255,255,0.1);
border-radius:0px;
font:16px Microsoft YaHei;
color:rgba(255,255,255,0.4);
border:none;
}

QToolButton:checked
{
background-color:rgba(0,145,255,1); 
border-radius:0px 0px 0px 2px;
font:16px Microsoft YaHei; 
color:rgba(255,255,255,1);
border:none;
}
</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="dnsBtn">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>250</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>16</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton
{
border:none;
background:#0E1417;
color:#B1C1D6;
font: 12px &quot;Microsoft YaHei&quot;;
}</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="icon">
    <iconset resource="../../CArmWorkStation.qrc">
     <normaloff>:/FunctionalWidget/FunctionalWidget/Review/Resource/triangle.png</normaloff>:/FunctionalWidget/FunctionalWidget/Review/Resource/triangle.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>7</width>
     <height>7</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="shpBtn">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>190</y>
     <width>22</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>22</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>22</width>
     <height>16</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton
{
border:none;
background:#0E1417;
color:#B1C1D6;
font: 12px &quot;Microsoft YaHei&quot;;
}</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="icon">
    <iconset resource="../../CArmWorkStation.qrc">
     <normaloff>:/FunctionalWidget/FunctionalWidget/Review/Resource/triangle.png</normaloff>:/FunctionalWidget/FunctionalWidget/Review/Resource/triangle.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>7</width>
     <height>7</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
  </widget>
  <widget class="QLabel" name="contsValue">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>25</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
border:none;
background:transparent;
color:#B1C1D6;
font: 12px &quot;Microsoft YaHei&quot;;
}</string>
   </property>
   <property name="text">
    <string>50</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="brtValue">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>91</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
border:none;
background:transparent;
color:#B1C1D6;
font: 12px &quot;Microsoft YaHei&quot;;
}</string>
   </property>
   <property name="text">
    <string>50</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="shpValue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>155</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
border:none;
background:transparent;
color:#B1C1D6;
font: 12px &quot;Microsoft YaHei&quot;;
}</string>
   </property>
   <property name="text">
    <string>50</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="dnsValue">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>219</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
border:none;
background:transparent;
color:#B1C1D6;
font: 12px &quot;Microsoft YaHei&quot;;
}</string>
   </property>
   <property name="text">
    <string>50</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="mtAuto">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>326</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>36</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>36</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
background:rgba(11,209,140,0.24);
border-radius:1px;
opacity:0.24;
color:rgba(11,209,140,1);
}</string>
   </property>
   <property name="text">
    <string>自动</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>contsSlider</zorder>
  <zorder>brtSlider</zorder>
  <zorder>label</zorder>
  <zorder>contsAuto</zorder>
  <zorder>label_3</zorder>
  <zorder>brtAuto</zorder>
  <zorder>shpSlider</zorder>
  <zorder>label_5</zorder>
  <zorder>dnsSlider</zorder>
  <zorder>label_8</zorder>
  <zorder>mt0Btn</zorder>
  <zorder>mt1Btn</zorder>
  <zorder>mt2Btn</zorder>
  <zorder>mt3Btn</zorder>
  <zorder>dnsBtn</zorder>
  <zorder>shpBtn</zorder>
  <zorder>contsValue</zorder>
  <zorder>brtValue</zorder>
  <zorder>shpValue</zorder>
  <zorder>dnsValue</zorder>
  <zorder>label_7</zorder>
  <zorder>mtAuto</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CArmBothwaySlider</class>
   <extends>QSlider</extends>
   <header>CArmBothwaySlider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../CArmTouchPanel/CArmTouchPanel.qrc"/>
  <include location="../../CArmWorkStation.qrc"/>
 </resources>
 <connections/>
</ui>
